recipes/argon2/all/test_package/argon2-test.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include "argon2/argon2.h"
```

Comment:
Upstream does not use argon2 subdirectory

```suggestion
#include "argon2.h"
```